pyscript.core/tests/integration/test_zz_examples.py

```diff
@@ -29,7 +29,7 @@ class TestExamples(PyScriptTest):
     @pytest.fixture()
     def chdir(self):
         # make sure that the http server serves from the right directory
-        ROOT.join("pyscriptjs").chdir()
+        ROOT.join("examples").chdir()
```

Comment:
I see, in that case, we perhaps don't need the `chdir` fixture at all
I say because the new `examples` folder is directly under the root of the repository

Else, we can also change the `goto` statements such as:

`self.goto('examples/hello_world.html')` --> `self.goto("hello_world.html")`

What do you recommend?

Reply:
right, I think we can just remove the chdir

Reply:
thanks!